lib/tasks/etl.rake

```diff
@@ -1,8 +1,8 @@
 namespace :etl do
-  desc "Run ETL master process for yesterday"
-  task master: :environment do
```

Comment:
I think we should keep the old `master` task around until the Puppet PR has been deployed so we don't end up with a period of time where the Jenkins job doesn't work. I think we can do something like this `task :master => :main`.

Reply:
I've made this change. Does it look right to you?

Reply:
I think it'll need to be two tasks, so something like this:

```rb
desc "Run ETL main process for yesterday"
task main: :environment do
  unless Etl::Master::MasterProcessor.process
    abort("Etl::Master::MasterProcessor failed")
  end
end

task master: :main  # master depends on main, but does nothing itself
```

Reply:
Oh right, yeah! I'll change that